objects.go

```diff
@@ -729,25 +729,49 @@ func NewHostRecord(
 }
 
 type RecordTXT struct {
-	IBBase `json:"-"`
-	Ref    string `json:"_ref,omitempty"`
-	Name   string `json:"name,omitempty"`
-	Text   string `json:"text,omitempty"`
-	Ttl    uint   `json:"ttl,omitempty"`
-	View   string `json:"view,omitempty"`
-	Zone   string `json:"zone,omitempty"`
-	Ea     EA     `json:"extattrs"`
-	UseTtl bool   `json:"use_ttl"`
+	IBBase  `json:"-"`
+	Ref     string `json:"_ref,omitempty"`
+	Name    string `json:"name,omitempty"`
+	Text    string `json:"text,omitempty"`
+	Ttl     uint32 `json:"ttl,omitempty"`
+	View    string `json:"view,omitempty"`
+	Zone    string `json:"zone,omitempty"`
+	Comment string `json:"comment"`
+	Ea      EA     `json:"extattrs,omitempty"`
+	UseTtl  bool   `json:"use_ttl,omitempty"`
 }
 
-func NewRecordTXT(rt RecordTXT) *RecordTXT {
-	res := rt
+func NewEmptyRecordTXT() *RecordTXT {
+	res := RecordTXT{}
 	res.objectType = "record:txt"
 	res.returnFields = []string{"extattrs", "name", "text", "view", "zone", "ttl", "use_ttl"}
```

Comment:
If you intend to update the value of 'comment' field, then you have to add it into this list.